### DISCUSSION TITLE
Case insensitive sorting

### DISCUSSION THREAD

**Discussion by DianaFAT-Moves** [directus/directus#20407] [Feature Requests]
### Summary

We are using Directus to store products and we created a user-friendly interface for multiple users to have the possibility to update the information about prices and availability. The table we use for this purpose is written in Vue.js. The products are different titles and can be written starting with an upper case or lower case letter. We can not change the way the products are written, but in this case, the alphabetical sorting is not working as expected. As the products that start with lowercase letters are placed at the end of the product list. Is there any possibility of changing this behavior? Our research showed that Directus uses only case-sensitive sorting and thus the API returns the product the same way they are sorted in Directus.

### Motivation

It is more convenient to work with data if the soring is caseßinsensitive as all the products that start with the same letter are shown as one list. Moreover, the products or their attributes wonʼt be lost in the list, if someone makes a typo.

### Detailed Design

Just make the soring case insensitive.

### Requirements List

case-insensitive sorting

### Drawbacks

-

### Alternatives

-

### Adoption Strategy

-

**Comment by tmaihoff** [comment]
I'm having the same issues. Is there a way to make sorting case insensitive?

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
